discord/interactions/connectCommand.ts

```diff
@@ -93,8 +120,13 @@ export const handleReconnectNetworkCommand = async (
     components: [],
   });
 
+  //verification process based on network type
+  const verificationUrl = isEthereumNetwork(networkName)
+    ? `${config.BASE_URL}/verify-eth/${guildId}/${userId}/${customLink}`
+    : `${config.BASE_URL}/verify/${guildId}/${userId}/${customLink}`;
+
   await interaction.followUp({
-    content: `Go to this link: ${config.BASE_URL}/verify/${guildId}/${userId}/${customLink} and verify your Starknet identity on network: ${networkName}!`,
+    content: `Go to this link: ${verificationUrl} and verify your Starknet identity on network: ${networkName}!`,
```

Comment:
You can write "and verify your identity" instead of "and verif your Starknet identity" because now we support other networks than starknet